tools/ci_build/build.py

```diff
@@ -1973,13 +1973,11 @@ def nuphar_run_python_tests(build_dir, configs):
 
 def tvm_run_python_tests(build_dir, configs):
     for config in configs:
-        if config == "Debug":
-            continue
         cwd = get_config_build_dir(build_dir, config)
         if is_windows():
             cwd = os.path.join(cwd, config)
-        dll_path = os.path.join(build_dir, config, "_deps", "tvm-build", config)
-        run_subprocess([sys.executable, "onnxruntime_test_python_tvm.py"], cwd=cwd, dll_path=dll_path)
+        python_path = str((Path(build_dir) / config / "_deps" / "tvm-src" / "python").resolve())
```

Comment:
I don't think Path is imported. Why not using os.path.join, it is used in other places of the same file?

Reply:
@xadupre, done